include/picongpu/particles/InitFunctors.hpp

```diff
@@ -48,10 +48,8 @@ namespace picongpu
     {
         /** call a functor
          *
-         * @tparam T_Functor unary lambda functor
-         *                   operator() must take two params
-         *                      - first: storage tuple
-         *                      - second: current time step
+         * @tparam T_Functor unary lambda functor, must be default-constructible and
+         *         operator() must take the current time step as the only parameter
          */
         template<typename T_Functor = bmpl::_1>
         struct CallFunctor
```

Comment:
I think `thisCallFunctor` should not be in `namespace particles`, as it has nothing to do with particles, and used to call general functors. But i'm not sure where to put it, so left here.

Reply:
Yes, let us move `CallFunctor` out of particles after this PR is merged.

Reply:
This change is implemented in #3608.